packages/block-library/src/term-template/variations.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { list, grid } from '@wordpress/icons';
+
+const createInnerBlocks = ( groupMetadata, groupStyle = {} ) => [
+	[
+		'core/group',
+		{
+			layout: {
+				type: 'flex',
+				orientation: 'horizontal',
+			},
+			style: {
+				spacing: {
+					blockGap: '0.5rem',
+				},
+				...groupStyle,
+			},
+			metadata: {
+				name: groupMetadata,
+			},
+		},
+		[
+			[
+				'core/paragraph',
+				{
+					metadata: {
+						name: __( 'Term Name' ),
+						bindings: {
+							content: {
+								source: 'core/term-data',
+								args: {
+									key: 'name',
+								},
+							},
+						},
+					},
+				},
+			],
+			[
+				'core/paragraph',
+				{
+					placeholder: __( '(count)' ),
+					metadata: {
+						name: __( 'Term Count' ),
+						bindings: {
+							content: {
+								source: 'core/term-data',
+								args: {
+									key: 'count',
+								},
+							},
+						},
+					},
+				},
+			],
+		],
+	],
+];
+
+const variations = [
+	{
+		name: 'list',
+		title: __( 'List' ),
+		description: __( 'Display terms in a list layout.' ),
+		attributes: { layout: { type: 'default' } },
+		isDefault: true,
+		icon: list,
+		scope: [ 'block', 'inserter' ],
+		innerBlocks: createInnerBlocks( __( 'Term Name with Count' ) ),
```

Comment:
Why do we need different metadata in these variations? How is `Term Name with Count` different than `Term Card`?

Reply:
`Term Card` is being used in the grid view as a container for the term details, whereas `Term Name with Count` is more simple inline piece of metadata that includes only the term name and the count. The variations are being reworked in https://github.com/WordPress/gutenberg/pull/71596, so this will probably change before being stabalised.